pages/DynamicPOC/index.tsx

```diff
@@ -65,15 +71,73 @@ export const getStaticProps: GetStaticProps = (context) => {
   }
 }
 
+let startingAnswer: FormAnswer
+
 const DynamicPOC: React.FC<{ questions: Question[] }> = ({ questions }) => {
   const startingQuestion: Question = questions[0]
 
-  const { formValues, updateFormValues } = useContext(FormCtx)
-
+  const [formValues, setFormValues] = useState<FormValues>(emptyFormValues)
   const [currentQuestion, setCurrentQuestion] = useState(startingQuestion)
   const [currentAnswer, setCurrentAnswer] = useState(startingAnswer)
   const [questionHistory, setQuestionHistory] = useState([startingQuestion])
   const [currentIndex, setCurrentIndex] = useState(0)
+  const [loaded, setLoaded] = useState(false)
+  const { data } = useSession()
```

Comment:
We should also extract the status here, so that we can redirect to signin if the user is signed out. Check out [this](https://next-auth.js.org/tutorials/securing-pages-and-api-routes) for an example, we only need to check for the `unauthenticated` case.